packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.css

```diff
@@ -11,11 +13,10 @@
 
 .recent-query__icon {
   margin-right: .5rem;
-  margin-left: .25rem;
 }
 
 .empty-list-message {
-  color: var(--sds-c-color-gray-8, #8E959D);
+  color: #8e959d;
```

Comment:
why you change this?

Reply:
According to [SLDS Design token doc](https://www.lightningdesignsystem.com/design-tokens/) this token is for internal use only right now so it isn't available to us. So it would always resolve to the fallback value anyway.

Reply:
Pretty disappointing tbh 😅